recipes/ginkgo/all/conanfile.py

```diff
@@ -109,7 +120,12 @@ def build(self):
         cmake.build()
 
     def package(self):
-        copy(self, "LICENSE", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        copy(
```

Comment:
Please, avoid changing the code format. It's not really mandatory following pep8, so it would cause only more code diff to review.

Reply:
I will maintain this package for the forseeable future, and my editor will try to format this every time I make a change, so not formatting would make my job much harder.

Reply:
But I'll insist, because formatting will result only in more time consumption for reviewers, is harder to review and will not change any behavior. Your PR only adds a new version, usually, this kind of PR consumes less than 10 lines of code in total. As you are probably using an IDE, it should easily configurable. So please, consider it for future PRs.

Reply:
I'll oblige, but I'm still surprised a project of this size does not have consistent formatting requirements.